### PR TITLE
Partial rgb support

### PR DESCRIPTION
**Pull request by hertzdude**

### CODE REVIEW

File: colorfield/validators.py [hunk 1]
```diff
@@ -19,3 +19,9 @@
 color_hexa_validator = RegexValidator(
     COLOR_HEXA_RE, _("Enter a valid hexa color, eg. #00000000"), "invalid"
 )
+
+
+COLOR_RGB_RE = re_compile("(#([\da-f]{3}){1,2}|(rgb|hsl)a\((\d{1,3}%?,\s?){3}(1|0?\.\d+)\)|(rgb|hsl)\(\d{1,3}%?(,\s?\d{1,3}%?){2}\))")
```
Comment (fabiocaccamo) [inline colorfield/validators.py:24]: This regular expression is too complex for doing just RGB colors validation.

**Review by fabiocaccamo — CHANGES_REQUESTED** [review]

**Review by fabiocaccamo — CHANGES_REQUESTED** [review]
1. If we add RGB colors support, we should add also RGBA colors support.
2. The PR doesn't pass tests.

**Comment by hertzdude** [comment]
OK, I'll do it within a couple of days.

**Comment by fabiocaccamo** [comment]
@hertzdude any update?

**Pull request closed without merging**